app/models/plugins/thermometer.rb

```diff
@@ -28,4 +29,57 @@ def liquid_data
   def name
     self.class.name.demodulize
   end
+
+  def update_goal
+    self.goal = self.determine_next_goal
+    self.save
+  end
+
+  protected
+  def goal_should_update
+    current_total >= self.goal
```

Comment:
In other words, once you're at 99,900 increase rather than waiting until you actually hit the 100,000? I don't much mind, unless not having an offset means people will sometimes see a signer count that is higher than the target. Otherwise, being the rare person who gets to get to the target exactly sounds kinda fun to me :)